pipeline/merger/src/test/scala/weco/pipeline/merger/PlatformMergerScenarioTest.scala

```diff
@@ -0,0 +1,37 @@
+package weco.pipeline.merger
+
+import weco.catalogue.internal_model.work.Work
+import weco.pipeline.merger.services.{DefaultPlatformMerger, Merger}
+
+class PlatformMergerScenarioTest extends MergerScenarioTest {
```

Comment:
Again marking out `PlatformMergerScenarioTest` and `MergerScenarioTest` - why is this one "Platform" - perhaps it should be `TEIMergerScenarioTest`?

Reply:
Hmm, there already is a `TEIMergerScenarioTest` ... I am finding it confusing because the scenarios in `PlatformMergerScenarioTest` seem to concern TEI merging exclusively.